go/analyzer/staticcheck/template/analyzer.go

```diff
@@ -0,0 +1,11 @@
+package template
+
+import "github.com/bazelbuild/rules_go/go/analyzer/staticcheck/util"
+
+var (
+	// Value to be added during stamping
+	name = "dummy-value-please-replace-using-x_defs"
```

Comment:
Does it make sense to leave this blank and then fail at build time if it's blank? There are tests to prevent this, but I really don't like magic strings.